src/packages/auth/functions.ts

```diff
@@ -129,11 +129,21 @@ export function decodeJwt(authorizationHeader: string | undefined): JwtDecode {
   }
 }
 
-export function generateJwt(username: string, defaultRole: string, allowedRoles: string[]): string | null {
+export function generateJwt(
+  username: string,
+  defaultRole: string,
+  allowedRoles: string[],
+  expiry?: string,
+): string | null {
+  if (typeof expiry === 'undefined') {
+    const { JWT_EXPIRATION } = getEnv();
+    expiry = JWT_EXPIRATION;
+  }
+
```

Comment:
Non-blocking: could clean this "optional default" logic up by either setting a fallback value for `expiry` in the function parms:

```suggestion
  expiry?: string = getEnv().JWT_EXPIRATION,
): string | null {
```

Reply:
Didn't realize you could call `getEnv()` in the function signature. Done!